styles/style.css

```diff
@@ -0,0 +1,190 @@
+ul{
+    list-style-type: none;
+}
+a{
+    text-decoration: none;
+}
+header{
+    padding: 0.5rem 0;
+}
+header div{
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0rem 6.6rem;
+}
+header div ul{
+    display: flex;
+}
+header div a{
+    font-weight: 700;
+    color: grey;
+    margin-right: 1.4rem;
+}
+header div a:hover{
+    color: orange;
+}
+.swiggy-logo{
+    max-width: 50px;
+}
+.nav-bar{
+    padding-left: 11.6rem;
+    background-color:tan;
+}
+.nav-bar nav{
+    display: flex;
+    flex-flow: row nowrap;
+    align-items: center;
+}
+.nav-bar nav ul li{
+    display: inline-block;
+    margin-right: 0.5rem;
+}
+.nav-bar nav ul li a{
+    color:grey;
+    font-size: 13px;
+    font-weight: 700;
+}
+.nav-bar nav ul li span{
+    color:grey;
+    font-size: 13px;
+}
+.nav-bar nav ul li a:hover{
+    color: black;
+}
+.banner{
+    background-color: rgb(41, 29, 56);
+    display: flex;
+    align-items: center;
+    padding-top: 1rem;
+    padding-bottom: 1rem;
+    padding-left: 14rem;
+}
+.banner img{
+    max-width: 300px;
+}
+.banner h1{
+    color: white;
+    font-size: 35px;
+    margin-left: 3rem;
+}
+main{
+    display: flex;
+    flex-flow: row nowrap;
+    min-height: calc(100vh - 440px);
+}
+.categories{
+    border-right: 0.5px solid grey;
+    overflow-y: auto;
+}
+.categories ul{
+    list-style-type: none;
+}
+.category-highlight{
+    color: orange;
+}
+.categories ul li{
+    margin-bottom: 0.5rem;
+    padding-left: 11rem;
+}
+.menu-items{
+    border-right: 0.5px solid grey;
+    padding-left: 4rem;
+}
+.menu-items .category-info{
+    margin-bottom: 3rem;
+}
+.menu-items .item-description{
+    margin-top: 3rem;
+    padding-bottom: 3rem;
+    border-bottom: 0.5px solid grey;
+}
+.item-description img{
+    max-width: 15px;
+    margin-bottom: 0.5rem;
+}
+.item-description .dishname{
+    font-weight: 700;
+}
+.menu-items div h3{
+    margin-bottom: 0rem;
+}
+.menu-items div p{
+    margin-top: 0rem;
+    color: grey;
+    font-weight: 500;
+}
+.cart{
+    padding-left: 3rem;
+}
+.cart div h3{
+    margin-bottom: 0rem;
+}
+.cart div p{
+    margin-top: 0rem;
+    color: grey;
+    font-weight: 500;
+}
+.cart div span{
+    display: flex;
+    flex-flow: row wrap;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: 0.5rem;
+    margin-bottom: 0;
+}
+.cart div span p{
+    margin: 0;
+    padding: 0;
+}
+.charges-description{
+    margin-top: 0;
+    font-size: 13px;
+}
+.checkout-btn{
+    width: 100%;
+    height: 2.7rem;
```

Comment:
Don't give height to this button, drive the height through padding. Please sync with Ayush on this